app/src/main/res/layout/fragment_updatebalance.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.UpdateBalanceFragment">
```

Comment:
We can talk about this in our meeting, but I think it would be cool to show the user the standing balance of their account as soon as they select it in the spinner. I know the ledger does all of this, but it was part of our original design to show how much a percent change the user is inputting when they update their balance. I think adding a new row maybe where you could show last balance and have a box next to it for the percent change would accomplish this.